Index: flevents-frontend/src/views/forms/EditAccount.vue
===================================================================
diff --git a/flevents-frontend/src/views/forms/EditAccount.vue b/flevents-frontend/src/views/forms/EditAccount.vue
deleted file mode 100644
--- a/flevents-frontend/src/views/forms/EditAccount.vue	
+++ /dev/null	
@@ -1,15 +0,0 @@
-<template>
-  <Heading text="Account bearbeiten" />
-  <CardBanner message="Im Endprodukt würden hier die entsprechenden Details des Accounts eingetragen sein. Außerdem wäre unten Felder zum Ändern des Passworts und Buttons zum Speichern und Verwerfen." />
-  <AccountForm />
-</template>
-
-<script setup lang="ts">
-import AccountForm from "@/components/AccountForm.vue";
-import Heading from "@/components/Heading.vue";
-import CardBanner from "@/components/CardBanner.vue";
-</script>
-
-<style scoped>
-
-</style>
Index: flevents-frontend/src/views/forms/CreateEvent.vue
===================================================================
diff --git a/flevents-frontend/src/views/forms/CreateEvent.vue b/flevents-frontend/src/views/forms/CreateEvent.vue
deleted file mode 100644
--- a/flevents-frontend/src/views/forms/CreateEvent.vue	
+++ /dev/null	
@@ -1,20 +0,0 @@
-<template>
-  <Heading text="Event erstellen" />
-  <EventForm
-    :back-route="'home.manage'"
-    :submit-route="'home.manage'"
-  />
-</template>
-
-<script lang="ts">
-import EventForm from "@/components/EventForm.vue";
-import Heading from "@/components/Heading.vue";
-export default {
-  name: "CreateAccount",
-  components: {Heading, EventForm}
-}
-</script>
-
-<style scoped>
-
-</style>
Index: flevents-frontend/src/views/event/addAccount.vue
===================================================================
diff --git a/flevents-frontend/src/views/event/addAccount.vue b/flevents-frontend/src/views/event/addAccount.vue
deleted file mode 100644
--- a/flevents-frontend/src/views/event/addAccount.vue	
+++ /dev/null	
@@ -1,91 +0,0 @@
-<script setup lang="ts">
-
-import {onBeforeMount, onUpdated, Ref, ref} from "vue";
-import {FleventsEvent} from "@/models/fleventsEvent";
-import {useRoute, useRouter} from 'vue-router';
-import axios from "axios"
-import Heading from "@/components/Heading.vue";
-const route = useRoute()
-const router = useRouter();
-const adress = ref("");
-const chips =  ref(new Array<any>());
-const tooltip = ref('');
-const event = ref({} as FleventsEvent)
-
-
-function remove(item: any){
-  chips.value.splice(chips.value.indexOf(item), 1)
-}
-
-// submit
-async function submit() {
-  try {
-    await axios.post("http://localhost:8082/api/mail/sendMail", { to: chips.value, cc: {}, bcc: {}, subject: `Anmeldung zu ${event.value.name} - TEST`, msgBody: `Sie wurden zu ${event.value.name} angemeldet.`, attachment: ""});
-    await router.push( {name: 'eventInfo', params: { uuid: event.value.uuid }} );
-  } catch (e) {
-    tooltip.value = "Das Event konnte nicht erstellt werden.";
-  }
-}
-
-onBeforeMount(async () => {
-  adress.value = route.params.id as string;
-  const response = await axios.get(`http://localhost:8082/api/events/${adress.value}`);
-  console.log(response);
-  response.status == 200 ? event.value = response.data : event.value = {} as FleventsEvent;
-})
-</script>
-
-<template>
-
-    <v-card>
-      <v-container>
-      <Heading :text="`${event.name} - Teilnehmer hinzufügen`"></Heading>
-      </v-container>
-      <v-form class="marignt" validate-on="submit" @submit.prevent="submit">
-        <v-container>
-          <v-combobox
-            v-model="chips"
-            chips
-            clearable
-            label="Eingeladene E-Mail-Adressen"
-            closable-chips
-            multiple
-            prepend-inner-icon="mdi-account"
-          >
-            <template v-slot:selection="{ attrs, select, selected }">
-              <v-chip
-                v-bind="attrs"
-                :model-value="selected"
-                closable
-                @click="select"
-                @click:close="remove"
-              >
-                <span>(interest)</span>
-              </v-chip>
-            </template>
-          </v-combobox>
-      </v-container>
-        <v-container class="d-flex flex-column flex-sm-row justify-end gap">
-          <v-btn
-            variant="outlined"
-            :to="{ name: 'events' }"
-          >
-            Verwerfen
-          </v-btn>
-          <v-btn
-            type="submit"
-            color="primary"
-          >
-            Teilnehmer hinzufügen
-          </v-btn>
-        </v-container>
-      </v-form>
-    </v-card>
-
-</template>
-
-<style scoped>
-.marignt{
-  margin-top: -3rem;
-}
-</style>
Index: flevents-frontend/src/views/forms/CreateAccount.vue
===================================================================
diff --git a/flevents-frontend/src/views/forms/CreateAccount.vue b/flevents-frontend/src/views/forms/CreateAccount.vue
deleted file mode 100644
--- a/flevents-frontend/src/views/forms/CreateAccount.vue	
+++ /dev/null	
@@ -1,17 +0,0 @@
-<template>
-  <Heading text="Account anlegen" />
-  <AccountForm />
-</template>
-
-<script lang="ts">
-import AccountForm from "@/components/AccountForm.vue";
-import Heading from "@/components/Heading.vue";
-export default {
-  name: "CreateAccount",
-  components: {Heading, AccountForm}
-}
-</script>
-
-<style scoped>
-
-</style>
Index: flevents-backend/flevents-backend.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<module version=\"4\">\r\n  <component name=\"FacetManager\">\r\n    <facet type=\"jpa\" name=\"JPA\">\r\n      <configuration>\r\n        <setting name=\"validation-enabled\" value=\"true\" />\r\n        <setting name=\"provider-name\" value=\"Hibernate\" />\r\n        <datasource-mapping>\r\n          <factory-entry name=\"entityManagerFactory\" />\r\n          <factory-entry name=\"flevents-backend\" />\r\n        </datasource-mapping>\r\n        <naming-strategy-map />\r\n      </configuration>\r\n    </facet>\r\n    <facet type=\"Spring\" name=\"Spring\">\r\n      <configuration />\r\n    </facet>\r\n  </component>\r\n</module>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/flevents-backend/flevents-backend.iml b/flevents-backend/flevents-backend.iml
--- a/flevents-backend/flevents-backend.iml	
+++ b/flevents-backend/flevents-backend.iml	
@@ -16,4 +16,8 @@
       <configuration />
     </facet>
   </component>
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
 </module>
\ No newline at end of file
